photography-site-demo.js/.config/jsonapi.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = Object.freeze({
+    jsonApiUrl: process.env.JSON_API_URL,
+    jsonApiConnectOptions: {
+        username: 'cassandra',
+        password: 'cassandra',
```

Comment:
The username and password should be passed from `JSON_API_AUTH_USERNAME` and `JSON_API_AUTH_PASSWORD` I think

Reply:
Yes, thanks!